main.go

```diff
@@ -31,9 +47,59 @@ func init() {
 }
 
 func main() {
+	log.Namespace = namespace
 	ctx := context.Background()
-	if err := service.Run(ctx, taxonomyRedirects); err != nil {
```

Comment:
So we should move the code from main to exist in service.Run(...) function, see my [dataset API pr](https://github.com/ONSdigital/dp-dataset-api/pull/215)